fed/mosaic-mapping/src/test/java/org/eclipse/mosaic/fed/mapping/ambassador/MappingAmbassadorTest.java

```diff
@@ -312,12 +312,12 @@ public void initializeWithMappingFile_scenarioVehicleRegistrationWithTypeDistrib
 
         ambassador.processInteraction(new ScenarioVehicleRegistration(0, "veh_0", new VehicleType("myCarDistribution")));
         assertVehicleRegistration(
-                "package.appA"
+                "package.appB"
```

Comment:
I feel like we should remove the initial stochastic selection in the `FixedOrderSelectior` since the `appA` has a weight of 0.6 and `appB` a weight of 0.4, one would expect the selection to start with `appA`. However, due to the stochastic nature of selection we start selection with `appB` here.

Reply:
@schwepmo With @rprotzmann we decided to remove randomness completely from the `FixedOrderSelector`. The first item is chosen according to the weights, if two or more items have the same weight, the first in the types list is selected. Now `fixedOrder = false` is the default behavior, since a natural stochastic selection is the more common use case and also what a user would usually expect. 

These test asserts here stay untouched though, since here now the StochasticSelector is used.